contracts/contracts/vault/VaultStorage.sol

```diff
@@ -236,8 +237,14 @@ contract VaultStorage is Initializable, Governable {
     /// @notice Mapping of withdrawal request indices to the user withdrawal request data
     mapping(uint256 => WithdrawalRequest) public withdrawalRequests;
 
+    /// @notice Used for OETH & superOETHb async withdrawal
```

Comment:
Would be nice if the comment stated the function of the variable. Smeth like: 
`@notice: Sets a minimum delay that is required to elapse between requesting async withdrawals and claiming the request. When set to 0 async withdrawals are disabled`

Reply:
Updated, thanks